### PR TITLE
Updated HowTo with IIH

### PR DESCRIPTION
**Pull request by j0hannesfr**
- updated Mindsphere Connector to IIH Mindsphere Sync
- reworked screenshots
- fixed some smaller grammar/spelling mistakes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,9 +59,9 @@ and forwarding them to MindSphere.
 - Established connection to 2 PLCs to acquire data with the Edge Device
 - Access to an Industrial Edge Management System (IEM)
 - Onboarded 3 Industrial Edge Devices (IEDs) on Industrial Edge Management
-- Installed System App Configurators on IEM (OPC UA Connector Configurator, IE Databus Configurator, Cloud Connector Configurator, IE MIndSphere Connector Configurator) 
+- Installed System App Configurators on IEM (OPC UA Connector Configurator, IE Databus Configurator, Cloud Connector Configurator) 
 - Installed apps on Southbound-Devices (OPC UA Connector, IE Cloud Connector, IE Databus, IE Flow Creator)
-- Installed apps on Northbound-Device (Data Service, IE Databus, IE Flow Creator, Energy Manager, IE MQTT Connector, IE Cloud Connector, IE MindSphere Connector)
+- Installed apps on Northbound-Device (Data Service, IE Databus, IE Flow Creator, Energy Manager, IE MQTT Connector, IE Cloud Connector, IIH Core, IIH Configurator, IIH registry Service, IIH Databus Gateway)
```
[the remark this answers is not in the document]
Comment (j0hannesfr) [reply README.md:64]: Removed Registry Service and Databus Gateway

File: docs/install_Device_Northbound.md [hunk 2]
```diff
@@ -87,130 +90,90 @@ Instead of manually configuring you can also import the configuration files:
...
-  
+  ![IIH_DatabusCredentials](graphics/IIH_DatabusCredentials.png)
 
-  ![IE_Dataservice6](graphics/IE_Dataservice6.png)
+2. Save the configuration
 
-13. Choose the data for Line1 and add them to the aspect. Do the same for Line2
-  
+3. Check if the device has a online connection in the "Home" tab. If the status is offline, please check your
+   network settings and make sure that everything is configured correctly.
```
[the remark this answers is not in the document]
Comment (j0hannesfr) [reply docs/install_Device_Northbound.md:124]: this was meant differently, I rephrased the sentence

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (j0hannesfr) [reply README.md:64]: Should I update all the screenshots showing the old names too?

[the remark this answers is not in the document]
Comment (Boehmi-TB) [reply README.md]: I would say so.

**Review by Boehmi-TB — APPROVED** [review]

**Pull request merged**